python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -562,6 +562,11 @@ def makeGraph(self, pipeline: Pipeline, args: SimpleNamespace) -> Optional[Quant
             if args.show_qgraph_header:
                 print(QuantumGraph.readHeader(args.qgraph))
         else:
+            task_defs = list(pipeline.toExpandedPipeline())
+            if args.mock:
```

Comment:
Is it not possible to have a test that uses mocks here?

Reply:
Yes, but I'd really like to start considering ci_middleware test coverage as sufficiently close to in-package test coverage that we avoid the duplication (of both effort and test-execution time).  In-package test coverage is still slightly preferable when it's just as easy to write a complete test there, but if I can only write a minimal test in-package that's wholly redundant with an easier-to-write and much more thorough test in ci_middleware, I'd prefer to get in the habit of just doing the latter.

Reply:
That's fine but makes it trickier for middleware users who aren't using the rest of the pipelines code.